tests/run-make/remap-path-prefix/rmake.rs

```diff
@@ -47,12 +47,51 @@ fn main() {
     }
 
     out_object.run();
-    invalid_utf8_contains("liblib.rmeta", "/the/aux/lib.rs");
-    invalid_utf8_not_contains("liblib.rmeta", "auxiliary");
+    rmeta_contains("/the/aux/lib.rs");
+    rmeta_not_contains("auxiliary");
     out_macro.run();
-    invalid_utf8_contains("liblib.rmeta", "/the/aux/lib.rs");
-    invalid_utf8_not_contains("liblib.rmeta", "auxiliary");
+    rmeta_contains("/the/aux/lib.rs");
+    rmeta_not_contains("auxiliary");
     out_diagobj.run();
-    invalid_utf8_contains("liblib.rmeta", "/the/aux/lib.rs");
-    invalid_utf8_not_contains("liblib.rmeta", "auxiliary");
+    rmeta_contains("/the/aux/lib.rs");
+    rmeta_not_contains("auxiliary");
+}
+
+//FIXME(Oneirical): These could be generalized into run_make_support
+// helper functions.
+fn rmeta_contains(expected: &str) {
+    // Normalize to account for path differences in Windows.
+    let haystack: Vec<u8> = fs_wrapper::read("liblib.rmeta")
+        .iter()
+        .map(|x| if *x == b'\\' { b'/' } else { *x })
+        .collect();
+    let needle = expected.as_bytes();
+    for i in 0..haystack.len() - needle.len() + 1 {
```

Comment:
This is hard to read and trust :/ What about something like this:
```rust
fn is_sub<T: PartialEq>(haystack: &[T], needle: &[T]) -> bool {
    haystack.windows(needle.len()).any(|c| c == needle)
}
```
(from https://stackoverflow.com/questions/47043167/does-rust-contain-a-way-to-directly-check-whether-or-not-one-vector-is-a-substr/47044053#47044053).

Or we can add https://docs.rs/bstr/latest/bstr/ to `run-make-support`.